pkg/client/endpoint.go

```diff
@@ -17,7 +17,7 @@ package client
 import (
 	"github.com/cilium/cilium/api/v1/client/endpoint"
 	"github.com/cilium/cilium/api/v1/models"
-	pkgEndpoint "github.com/cilium/cilium/pkg/endpoint"
+	pkgEndpointID "github.com/cilium/cilium/pkg/endpoint/id"
```

Comment:
Merge this commit with the id package commit in order not to break git bisecting as the previous commit alone will not compile.